### ISSUE TITLE
Indention is incorrect when using colon-statement separator

### ISSUE THREAD

**Issue by shortad** [rokucommunity/brighterscript-formatter#92]
Roku Documentation: https://developer.roku.com/en-ca/docs/references/brightscript/language/statement-summary.md

### Actual Indentation for Colon-Statement Syntax
See line **706**, incorrectly knocking the indent back when using colon-statement syntax

![image](https://github.com/rokucommunity/brighterscript/assets/102538093/a4b113cd-c11d-46dc-958e-b88a6059bc6c)

### Expected Indentation for Colon-Statement Syntax
This example is taken _before_ the BS code formatter modified the indentation.

![image](https://github.com/rokucommunity/brighterscript/assets/102538093/09cddee7-7aac-4fec-b5f2-e564b0c7db71)

### Correct Indentation Shown With **If-Then** Syntax
See line **706**, correctly indented when using "if then" syntax

![image](https://github.com/rokucommunity/brighterscript/assets/102538093/1299c1ba-79fd-4240-bad9-256ce7d047ad)

**Comment by TwitchBronBron** [comment]
Yeah, this is one of the deficiencies in the formatter. Thanks for raising this. And also, don't hold back on raising the issues. There's only a few of us to work on all this stuff, but it's still way more valuable to know about the issues, so keep 'em coming!

**Comment by shortad** [comment]
After things settle a bit for me (_so close_ to being finished with this giant work project) I'll see if I can get these repos set up on my local so I can make actionable suggestions/PRs 🫡

**Comment by TwitchBronBron** [comment]
That would be great! Here's a small writeup on how to get started contributing to the vscode extension and several of its associated packages:

https://rokucommunity.github.io/vscode-brightscript-language/contributing.html#the-easy-way
